package.json

```diff
@@ -37,16 +43,20 @@
   },
   "main": "src/index.js",
   "scripts": {
-    "sst": "mocha test/acceptance/index.test.js --timeout 15000",
-    "loc": "sloc src/ examples/",
-    "test": "npm run lint",
-    "lint": "eslint .",
-    "lint:fix": "eslint . --fix",
-    "build-dev": "webpack --config webpack/webpack.config.js",
     "build-all": "webpack --config webpack/webpack.config.js && webpack --config webpack/webpack.min.config.js",
-    "build:watch": "webpack -w --config webpack/webpack.config.js",
-    "build-docs": "rm -rf docs/public; jsdoc --configure config/jsdoc/public-conf.json",
+    "build-dev": "webpack --config webpack/webpack.config.js",
```

Comment:
I think we don't need these commands `build-all`, `build-dev` since we have `build:watch` and `build`

Reply:
Some kind of `build` is needed for the integration tests and for ghpages deployment.